gomint-server/src/main/java/io/gomint/server/network/packet/PacketResourcePackStack.java

```diff
@@ -19,6 +19,7 @@
     private List<ResourcePack> resourcePackEntries;
 
     private boolean experimental;
+    private String gameVersion = Protocol.MINECRAFT_PE_NETWORK_VERSION;
```

Comment:
Please don't copy over the string, you can directly write the constant with writeString